documentation/features/dashboard/README.md

```
@@ -136,7 +136,7 @@ This allows for faster editing of widgets or exchanging different dashboard widg
 
 ## Access control
 
-Access to dashboards can be configured through access control. More information about access control can be found [here](https://docs.valtimo.nl/features/access-control).
+Access to dashboards can be configured through access control. More information about access control can be found [here](../access-control/README.md).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)